docs/guide/docs/tutorials/proactive.md

```diff
@@ -42,8 +42,20 @@ if (event.type === 'proactive-trigger') {
 
 > **Tip**: Use `event.setFlag(bp.IO.WellKnownFlags.SKIP_DIALOG_ENGINE, true)` to tell the dialog engine to skip the event processing. This is useful when your event is not a user message.
 
+## Webchat events
+
+There's currently 3 events that can be catched in your page :
+
+| name             | Description                                                  |
+| ---------------- | ------------------------------------------------------------ |
+| `webchatLoaded`  | Triggered when the webchat is loaded and ready to be openned |
+| `webchatOpenned` | Triggered when the webchat button bubble is clicked          |
+| `webchatHidden`  | Triggered when the webchat close button is clicked           |
```

Comment:
Can you change Openned to Opened ? (and at other place where there is the typo)
Also, to be consistent, I would keep webchatClosed instead of webchatHidden